### DISCUSSION TITLE
The installation script installs an outdated version on a Fedora instance

### DISCUSSION THREAD

**Discussion by chengguangnan** [netdata/netdata#17813] [Q&A] [answered]
I have two Fedora instances, all running with Fedora 40.

On one instance, the script installed "v1.45.5", but another instance it installed "v1.45.0-526-nightly" and the web UI isn't working.

I'm not sure how to install the latest v1.45.5 on the second Fedora instance and I hope that will make the web UI working.

**Comment by chengguangnan** [comment]
I found that the working instance have "/etc/yum.repos.d/netdata.repo" but the not working instance doesn't. I copied the netdata.repo file over and installed v1.45.5 on the second machine.

**Comment by ilyam8** [comment, marked as answer]
Hi. v1.45.0-526-nightly is the latest nightly release (see [the badges here](https://github.com/netdata/netdata#monitor-your-servers-containers-and-applicationsin-high-resolution-and-in-real-time): latest stable and nightly). The nightly version numbering is confusing, yes. It always has `0` as a patch in the version string. See https://github.com/netdata/netdata/issues/14855

> the script installed "v1.45.5", but another instance it installed "v1.45.0-526-nightly" 

Stable vs Nightly. You added `--stable-channel` when installing on the first instance.
